### PR TITLE
docs(readme): Add readme content

### PR DESCRIPTION
**Pull request by Daedaleus**
- Small description changes
- Basic image for overview and detail view
- Example settings.toml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,56 @@
...
+url = "https://your.jira.url"
+username = "your-username"
+password = "your-password"
+jql = "assignee = currentUser() and statusCategory != Done"
+
+[[provider]]
+name = "Tasks"
+type = "taskwarrior"
+query = "status:pending"
+```
```
[the remark this answers is not in the document]
Comment (maxjoehnk) [reply README.md:66]:
```toml
[[provider]]
type = "upsource"
url = "https://your-upsource-instance"
token = "auth token"
# optional, default is "state: open"
query = "an upsource query"
```

**Review by maxjoehnk — APPROVED** [review]
LGTM when upsource example is added.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Daedaleus) [reply README.md:73]: Done

**Comment by maxjoehnk** [comment]
I just noticed, I think it would be nice if you could update the screenshot of the detail view as it looks way better now :D

**Comment by Daedaleus** [comment]
New imgs created

**Pull request merged**